datasets/squad.py

```diff
@@ -0,0 +1,149 @@
+import json
+import pathlib
+from transformers import BertTokenizer
+import numpy as np
+from extra.utils import download_file
+
+BASEDIR = pathlib.Path(__file__).parent.parent / "datasets/squad"
+download_file("https://rajpurkar.github.io/SQuAD-explorer/dataset/dev-v1.1.json", BASEDIR / "dev-v1.1.json")
```

Comment:
I'm not sure if this is the best place to put the "downloader". It feels kinda weird.

Reply:
I'd put it in iterate, it shouldn't trigger on import.